gm4_lightning_in_a_bottle/data/gm4_lightning_in_a_bottle/loot_tables/entities/charged_creeper.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Why does this exist?

Reply:
idk, I just copied Orbis?

Reply:
Couldn't you just leave it without a loot table and it'd drop the default creeper loot table anyways?

Reply:
This is useful in case we want to change what the entity drops. If we decide the rates should decrease or drop different items, we can change the loot table instead of the spawner, since we won't be able to change the spawner's NBT.